### ISSUE TITLE
Error in Unity topic

### ISSUE THREAD

**Issue by CeciAc** [MicrosoftDocs/visualstudio-docs#1424]
@pslusarczyk commented on [Thu Aug 02 2018](https://github.com/MicrosoftDocs/visualstudio-docs.pl-pl/issues/2)

Wbrew temu, co podaje dokumentacja, w Unity Hub nie ma możliwości zainstalowania VS. Na liście komponentów do najnowszej wersji 2018.2 nie ma sekcji Dev Tools ani żadnej innej, która by zawierała VS. Pozdrawiam

---
#### Szczegóły dokumentu

⚠ *Nie edytuj tej sekcji. Jest ona wymagana dla docs.microsoft.com ➟ Dołączanie problemu w usłudze GitHub.*

* ID: 87f1198e-ec10-c455-9c19-c3b1f042d9c0
* Version Independent ID: 13b99f02-23dc-efcb-43a7-092f74e08290
* Content: [Wprowadzenie do narzędzi Visual Studio Tools for Unity](https://docs.microsoft.com/pl-pl/visualstudio/cross-platform/getting-started-with-visual-studio-tools-for-unity)
* Content Source: [docs/cross-platform/getting-started-with-visual-studio-tools-for-unity.md](https://github.com/MicrosoftDocs/visualstudio-docs.pl-pl/blob/live/docs/cross-platform/getting-started-with-visual-studio-tools-for-unity.md)
* Product: **visual-studio-dev15**
* GitHub Login: @dantogno
* Microsoft Alias: **v-davian**

---

@LisandroSu commented on [Thu Aug 02 2018](https://github.com/MicrosoftDocs/visualstudio-docs.pl-pl/issues/2#issuecomment-410087125)

Witaj @pslusarczyk 

Dziękujemy za opinię. 

Aktywnie przeglądamy Twoje komentarze i wkrótce się do Ciebie odezwiemy. 

Pozdrawiamy, 
Międzynarodowy zespół witryny Microsoft DOCS

**Comment by CeciAc** [comment]
Hi team, 
I'm copying this here since the user reports an inaccuracy in the documentation.
Roughly translation:
_"Contrary to what the documentation provides, it is not possible to install VS in the Unity Hub. In the list of components to the latest version 2018.2 there is no Dev Tools section or any other that would contain VS. Regards"_

best

**Comment by faraodevelopment** [comment]
Problem is that the plugin for Visual Studio 2019 isn't working with Unity 2019 (don't know if it works with Unity 2018).
I have reinstalled VS 2017, which works fine right out of the box. I have tried hacks to move dlls into the 2019 environment, but it doesn't work. The Tools for Unity is broken on VS2019.

**Comment by conceptdev** [comment]
I'll pass this on to the team, but it's also worth using Visual Studio [**Help > Report a problem**](https://docs.microsoft.com/visualstudio/ide/how-to-report-a-problem-with-visual-studio?view=vs-2019) so that system information and logs can be provided for troubleshooting.

**Comment by therealjohn** [comment]
Hi! The Unity Hub won't show Visual Studio if it's already installed on the machine. The Hub also does not currently offer VS 2019 - that will change in a future update. 

There is a problem with some versions of Unity 2019.2 and .3 with VS 2019 caused by the new vs.ide Unity Package that is used in those versions. We worked with Unity to identify that problem and those packages were changed. 2019.1 is unaffected.
